### DISCUSSION TITLE
Adding support for SearXNG

### DISCUSSION THREAD

**Discussion by grimaldiss** [ALwrity/ALwrity#118] [Ideas]
How about adding support for SearXNG

**Comment by AJaySi** [comment]
Hello,

ALwrity target audience are content creators & digital marketing professionals, who are non tech. Our mission is to empower, not overwhelm, them with AI. The tech-stack needs to be completely abstracted to avoid any learning curve.

We have avoided integrating tech that is hard to install Or complicated to get APIs Or lacks windows support. searx, ollama(has improved), local models, google APIs etc.

Of course, giving them SaaS is the easiest UI for them, so we are now working/struggling with next.js alwrity app.
For developers, the libraries is arranged as legos to create their own workflow/pipelines, FastAPIs coming soon.

Privacy is great concern and Ollama, plausible, searxng are on the card.
Kindly, correct me i feel searxng is *difficult to get started & our target audience prefers windows(hit)

Thanks a lot for the great pointer, we will keep it open and get to it.
BTW: We are working/struggling with Integrating social media platforms & AI editor (kill me now)
